### DISCUSSION TITLE
차단 목록 추가 요청

### DISCUSSION THREAD

**Discussion by dragonlee64** [List-KR/community#46] [AdGuard]
https://accentia.dict.naver.com

네이버 사전 이용 중에 자꾸 거슬려서 차단 부탁드립니다. 이것만 차단하면 이것과 관련된 다른 링크도 전부 차단되는 것이죠?

**Comment by piquark6046** [comment]
이거 말씀하시는 건가요?

<details><summary>스크린샷</summary>

![Screenshot from 2023-11-17 09-01-54](https://github.com/List-KR/community/assets/98787049/8dfa5c16-9150-4aaa-8bc6-f331d8dcae99)

</details> 

이게 아니라면, 스크린샷과 함께 알려주세요.

**Reply by dragonlee64** [reply]
네 맞습니다. 그런데, 네이버 사전 웹사이트 버전에서 해당 광고를 차단하는 방법은 제가 알고 있으나, 네이버 사전 '어플' 에서는 차단하는 방법을 모르겠습니다. 네이버 사전 어플 광고 문제로 인해 제가 몇 달 전에 https://github.com/List-KR/List-KR/issues/755 이 글을 남겼었고 그때 잘 해결해주셨으니 참고 차원에서 링크 남겨드립니다.

**Reply by piquark6046** [reply]
저 이미지는 웹 사이트와 같은 도메인에서 불러와서 루팅하시지 않는 이상 웹에서만 차단이 가능합니다.

**Reply by dragonlee64** [reply]
그렇군요, 저번과는 다른 경우인가 보네요. 저거 하나를 막기 위해 루팅하는 건 힘들 것 같아서 포기하려고 합니다... 답변 감사드립니다.
